src/decode/xz.rs

```diff
@@ -3,10 +3,11 @@
 use crate::decode::lzma2::Lzma2Decoder;
 use crate::decode::util;
 use crate::error;
-use crate::xz::{footer, header, CheckMethod, StreamFlags};
+use crate::xz::{
+    crc::{CRC32, CRC64},
+    footer, header, CheckMethod, StreamFlags,
+};
```

Comment:
The implicit style so far was to format imports with `imports_granularity="Module"` (https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#module). Could you format the new imports accordingly?

```rust
use crate::xz::crc::{CRC32, CRC64};
use crate::xz::{footer, header, CheckMethod, StreamFlags};
```

I'll separately look into how to update the format lint check to use this (unstable) rustfmt option.